### ISSUE TITLE
net.pms.util.jna.JNATypesTest fails on api-3-4e4787b

### ISSUE THREAD

**Issue by ajkessel** [UniversalMediaServer/UniversalMediaServer#2151]
Attempting to build  api-3-4e4787b branch, I get the following test failure. Googling has not turned up any clues.
`-------------------------------------------------------------------------------
Test set: net.pms.util.jna.JNATypesTest
-------------------------------------------------------------------------------
Tests run: 5, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.056 s <<< FAILURE! - in net.pms.util.jna.JNATypesTest
net.pms.util.jna.JNATypesTest.testFixedCharArrayByReference  Time elapsed: 0.004 s  <<< ERROR!
java.lang.NoSuchMethodError: java.nio.CharBuffer.flip()Ljava/nio/CharBuffer;
        at net.pms.util.jna.JNATypesTest.testFixedCharArrayByReference(JNATypesTest.java:188)`

**Comment by ajkessel** [comment]
Switching from OpenJDK to Oracle JDK 14.0.1 gets past this error -- is OpenJDK not supported for building UMS?

**Comment by SubJunk** [comment]
@ajkessel we currently use AdoptOpenJDK, I haven't tested with OpenJDK. Does it work for you with Adopt?

**Comment by ajkessel** [comment]
Yes, I just built with Adopt and it worked better than OpenJDK or Oracle JDK--it actually created the distribution tarball which neither of the others did. The only glitch was that it failed to build (via mvn) until I enabled X11 forwarding. The box I'm building this on is headless. Seems like you shouldn't need graphics display to build the tarball, presumably this is something that could be disabled somewhere in the build process.

**Comment by SubJunk** [comment]
@ajkessel yeah that seems like it could use some improvement. If you fix it we would welcome a Pull Request for the fix :)

**Comment by ajkessel** [comment]
I thought maybe adding PMS.forceHeadless() into LanguageSelectionClassTest() might do it, but it doesn't seem to have any effect.

**Comment by SubJunk** [comment]
@ajkessel I have little experience with that situation so I don't think I'll be much help, sorry
